package.json

```diff
@@ -56,10 +63,11 @@
     ]
   },
   "scripts": {
+    "prepare": "oclif-dev manifest && oclif-dev readme",
     "postpack": "rm -f oclif.manifest.json",
+    "test": "nyc mocha test/*/*.test.js",
     "posttest": "eslint .",
-    "prepare": "oclif-dev manifest && oclif-dev readme",
-    "test": "echo 'No tests specified'",
-    "version": "oclif-dev readme && git add README.md"
+    "report-coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
+    "clean": "for d in test-projects/*; do pushd $d; rm -rf package-lock.json node_modules dist build .nui .asset-compute; popd; done"
```

Comment:
`npm run clean` won't run on Windows...
Error: `d was unexpected at this time.`

Reply:
Yes, I know but it's an optional script for the developer working on new unit tests/test projects that was helpful for me. Not required or called by any build step. The unit tests themselves ensure a clean slate at the beginning.